src/reusable-blocks/class-reusable-blocks-basket-element.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace WPML\PB\Gutenberg;
+
+class Reusable_Blocks_Basket_Element {
```

Comment:
We don't have a specific object object to hold the basket items. I created this class which is quite similar with `WPML_TM_Translation_Batch_Element` and allows to share a good part of the code.

Reply:
It might be a good idea to extract the constructor and the getters into a parent (abstract, if needed) class so that both `\WPML_TM_Translation_Batch_Element` and ´\WPML\PB\Gutenberg\Reusable_Blocks_Basket_Element´ would share the code instead of duplicating it.

Reply:
@andreasciamanna, at the beginning I extended `Reusable_Blocks_Basket_Element` from `\WPML_TM_Translation_Batch_Element` but this was a bit hacky. Then later I add troubles with tests because `\WPML_TM_Translation_Batch_Element` is in another project.

If I start sharing code for this simple class, I will still have to create a stub in this project for testing. That's why I opted for this solution.

Reply:
Ok.

We can address that in the future then.